pkg/chunk/grpc/gRPC.proto

```diff
@@ -0,0 +1,111 @@
+syntax = "proto3";
+
+package grpc;
+
+service grpc_store {
+    //index-client
+    rpc BatchWrite(WriteBatch) returns (Nothing);
+    rpc QueryPages(IndexQuery) returns (stream ReadBatch);
+
+    //storage-client
+    rpc PutChunks(Chunks) returns (Nothing);
+    //In GetChunks rpc request send buf as nil
+    rpc GetChunks(Chunks) returns (stream Chunks);
+    rpc DeleteChunks(ChunkID) returns (Nothing);
+
+    //table-client
+    rpc ListTables(Nothing) returns (ListTablesResponse);
+    rpc CreateTable(TableDesc) returns (Nothing);
+    rpc DeleteTable(TableName) returns (Nothing);
+    rpc DescribeTable(TableName) returns (DescribeTableResponse);
+    rpc UpdateTable(UpdateTableRequest) returns (Nothing);
+
+    //used by storage-client & index-client
+    rpc Stop(Nothing) returns (Nothing);
+}
+
+message Chunks {
+    repeated Chunk chunks = 1;
+}
+
+message Chunk {
+    bytes buf         = 1;
```

Comment:
I would name it more clearly. This is the encoded chunk, so why not calling it `encoded`?